fr/index.md

```diff
@@ -0,0 +1,311 @@
+
+Quelques grands points de départ sont consultables [ici](https://modelviewculture.com),
+[ici](http://www.ashedryden.com/archive), et [ici](http://geekfeminism.wikia.com),
+et vous pouvez trouver plus de ressources à la fin de cette page.
+
+Si étudier ces sources ne vous convainc toujours pas, vous pourriez alors ne pas être la bonne personne pour faire marcher un groupe d’étude réservé aux femmes.  
+
+## Soyez humble: vous n'êtes pas le centre d'attention
+
+Quoi que vous fassiez, restez concentré sur vos élèves. Elles sont le groupe d'étude, pas vous. 
+
+Votre travail est de les aider à apprendre tout en s'amusant, de rester motivé, enthousiaste et content de leurs progrès. 
+
+Soyez humble: vous n'êtes pas le centre d'attention.
+
+
+## Prenez des pauses
+
+Pendant les ateliers, il est important de prendre des pauses. 
+
+Vos étudiantes seront plus rapidement fatiguées que vous puisqu'elles découvrent de nouvelles choses. Quoi qu’il en soit, la plupart du temps, être mentor est aussi très fatiguant. 
+
+Faites en sorte de garder un oeil sur le temps que vous mettez à expliquer un nouveau concept ou à les laisser résoudre un problème par elles-même. Prenez des pauses fréquemment. 
+
+Pendant les pauses, encouragez tout le monde à fermer les pc, aller faire un tour, discuter et prendre une collation. 
+
+
+## Posez des questions sur les expériences précédentes
+
+Souvent, les étudiantes qui prennent part à nos groupes de débutantes n'ont aucune expérience dans la programmation. 
+De temps en temps, elles ont quelques notions de HTML/CSS, ont déjà fait des modifications sur un blog WordPress ou ont hacké un mode dans un jeu vidéo. 
+D'autres sont ou ont été étudiantes en science de l'informatique, mais trouvent les cours de Java ou d'Haskell qu'elles ont suivi tellement nuls qu'elles recherchent une meilleure alternative. 
+
+À votre tout premier atelier, et à chaque fois qu'une nouvelle personne rejoint le groupe, demandez les expériences et compétences déjà acquises, ainsi que la motivation qui les pousse à rejoindre le groupe. Cela vous donnera quelques clés pour savoir comment commencer et quels points sont familiers. 
+
+
+## Restez aussi simple que possible
+
+Si vous pensez pouvoir mentorer un groupe de débutantes, vous êtes probablement un développeur expérimenté. 
+
+Gardez à l'esprit qu'être un bon développeur et être un bon professeur et/ou mentor sont deux choses bien distinctes qui requièrent différentes compétences. 
+
+Quelque soient vos propres centres d'intérêt, vos outils préférés, le style de programmation que vous aimez : ce sont des conclusions que vous avez fait après avoir mis des années à essayer différents outils, à rater certains projets, à apprendre de nouveaux types d’architectures et à recommencer autrement. 
```

Comment:
I keep all of it except the S of "d'intérêts" which seems to be singular :) 
thanks for the great improvement on that !